packages/components/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@platzily-ui/components",
+  "version": "0.2.0",
```

Comment:
The version should be 0.1.2, this component is a minur feature, when we get our first release we could say 0.1.x is completed, then when we add more composed features we could change to 0.2.0 or 0.2.x